### PR TITLE
Clevertap updates

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Updated doc to add conversion of objects and arrays to string

### CODE REVIEW

**Review by sauravlal15 — APPROVED** [review]

File: docs/destinations/streaming-destinations/clevertap.mdx [hunk 1]
```diff
@@ -625,6 +625,10 @@ rudderanalytics.page("Cart", "Cart Viewed", {
 })
 ```
 
+<div class="infoBlock">
+CleverTap does not support nested objects or arrays for custom attributes in the <code class="inline-code">page</code> events. Hence, Rudderstack converts the nested objects or arrays to string before sending them to CleverTap.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/clevertap.mdx:629]:
```suggestion
CleverTap does not support nested objects or arrays for custom attributes in the <code class="inline-code">page</code> events. Hence, RudderStack converts the nested objects or arrays into strings before sending them to CleverTap.
```

File: docs/destinations/streaming-destinations/clevertap.mdx [hunk 2]
```diff
@@ -638,6 +642,10 @@ A sample `screen` call looks like the following code snippet:
 
 In the above snippet, RudderStack captures all the information related to the screen being viewed, along with any additional info associated with that screen view event. In CleverTap, the above `screen` call will be shown as - **"Screen Viewed: `Sample Screen Name` "** along with the properties.
 
+<div class="infoBlock">
+CleverTap does not support nested objects or arrays for custom attributes in the <code class="inline-code">screen</code> events. Hence, Rudderstack converts the nested objects or arrays to string before sending them to CleverTap.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/clevertap.mdx:646]:
```suggestion
CleverTap does not support nested objects or arrays for custom attributes in the <code class="inline-code">screen</code> events. Hence, RudderStack converts the nested objects or arrays into strings before sending them to CleverTap.
```

File: docs/destinations/streaming-destinations/clevertap.mdx [hunk 3]
```diff
@@ -655,15 +663,11 @@ rudderanalytics.track("Checked Out", {
...
-
 To set a specific value to the <code class="inline-code">screen</code> or <code class="inline-code">track</code> type event, you need to pass the <code class="inline-code">event</code> related property in the <code class="inline-code">properties</code> field.
-
 </div>
 
 <div class="infoBlock">
-
-Note: For <code class="inline-code">track</code>, <code class="inline-code">page</code> and <code class="inline-code">screen</code> events CleverTap does not support arrays or nested objects for custom event properties.
-
+CleverTap does not support nested objects or arrays for custom attributes in the <code class="inline-code">track</code> events. Hence, Rudderstack converts the nested objects or arrays to string before sending them to CleverTap.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/clevertap.mdx:670]:
```suggestion
CleverTap does not support nested objects or arrays for custom attributes in the <code class="inline-code">track</code> events. Hence, RudderStack converts the nested objects or arrays into strings before sending them to CleverTap.
```

File: docs/destinations/streaming-destinations/clevertap.mdx [hunk 4]
```diff
@@ -783,9 +787,7 @@ If a user already exists, the new values will be updated for that user. Rudderst
 </div>
 
 <div class="infoBlock">
-
-Note: For <code class="inline-code">identify</code> events CleverTap does not support nested objects for user's <code class="inline-code">traits</code>.
-
+CleverTap does not support nested objects or arrays for custom attributes in the <code class="inline-code">identify</code> events. Hence, Rudderstack converts the nested objects or arrays to string before sending them to CleverTap.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/clevertap.mdx:790]:
```suggestion
CleverTap does not support nested objects or arrays for custom attributes in the <code class="inline-code">identify</code> events. Hence, RudderStack converts the nested objects or arrays into strings before sending them to CleverTap.
```

**Review by ameypv-rudder — APPROVED** [review]
LGTM overall - made some minor suggestions

**Pull request merged**